include/deal.II/physics/elasticity/kinematics.h

```diff
@@ -223,7 +223,7 @@ namespace Physics
        * gradient).
        * The result is expressed as
        * @f[
-       *  \mathbf{d} := \frac{1}{2} \left[ \mathbf{l} + \mathbf{l}^{T} \right] \, .
```

Comment:
don't you want to put `,` there instead of `.`? Don't recall whether there are punctuations in other formulas.